source/common/config/grpc_mux_impl.cc

```diff
@@ -316,7 +316,7 @@ void GrpcMuxImpl::queueDiscoveryRequest(absl::string_view queue_item) {
     api_state.pending_ = true;
     return; // Drop this request; the unpause will enqueue a new one.
   }
-  request_queue_->emplace(std::string(queue_item));
+  request_queue_->emplace(absl::string_view(queue_item));
```

Comment:
Since `queue_item` is already `absl::string_view` type, this cast is not necessary. Same above.

Reply:
let me change it then.